layouts/partials/widgets/contact.html

```diff
@@ -11,13 +11,13 @@
   <div class="col-xs-12 col-md-8">
     {{ with $page.Content }}<p>{{ . | markdownify }}</p>{{ end }}
 
-    <ul class="fa-ul">
+    <ul class="fa-ul" itemscope>
 
       {{ with $.Site.Params.email }}
       <li>
         <i class="fa-li fa fa-envelope fa-2x" aria-hidden="true"></i>
         <span>
-        {{- if $autolink }}<a href="mailto:{{ . }}">{{ . }}</a>{{ else }}{{ . }}{{ end -}}
+        {{- if $autolink }}<a href="mailto:{{ . }}" id="person-email" itemprop="email">{{ . }}</a>{{ else }}<span id="person-email" itemprop="email">{{ . }}</span>{{ end -}}
```

Comment:
Can the `id` and `itemprop` work if it goes in the above `span` on line 19 instead? If that works, it would make the code cleaner..